srtp/srtp.c

```diff
@@ -2074,6 +2060,154 @@ srtp_remove_stream(srtp_t session, uint32_t ssrc) {
 }
 
 
+srtp_err_status_t
+srtp_update(srtp_t session, const srtp_policy_t *policy) {
+  srtp_err_status_t stat;
+
+  /* sanity check arguments */
+  if ((session == NULL) || (policy == NULL) || (policy->key == NULL))
+    return srtp_err_status_bad_param;
+
+  while (policy != NULL) {
+    stat = srtp_update_stream(session, policy);
+    if (stat) {
+      return stat;
+    }
+
+    /* set policy to next item in list  */
+    policy = policy->next;
+  }
+  return srtp_err_status_ok;
+}
+
+
+srtp_err_status_t
+srtp_update_stream(srtp_t session, const srtp_policy_t *policy) {
```

Comment:
srtp_update_stream is called from srtp_update which is called in the test and therefore this function is tested.
As to if it should be public or not is debatable, I was trying once again to follow the rest of the srtp api that has both the create function where a list of policies can be used or else the add_stream function that operates on just one policy. Could make it so you can only update one stream at a time therefore only having one function but it is not consistent with rest of api.